packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -0,0 +1,189 @@
+      '--terra-status-view-message-font-size': '24px',
+      '--terra-status-view-message-padding-top': '10px',
+      '--terra-status-view-message-text-align': 'left',
+      '--terra-status-view-divider-padding-top': '10px',
+      '--terra-status-view-buttons-flex-direction-small-width': 'column',
+      '--terra-status-view-buttons-padding-top': '10px',
+      '--terra-status-view-button-flex': 'auto',
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Displays an error status view with the glyph being hidden', () => {
+    before(() => browser.url('/#/tests/status-view-tests/glyph-hidden'));
+
+    shouldTheme({
+      '--terra-status-view-padding-bottom': '10px',
+      '--terra-status-view-padding-top': '10px',
+      '--terra-status-view-title-color': 'purple',
+      '--terra-status-view-title-font-family': 'Impact',
+      '--terra-status-view-title-font-size': '36px',
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Displays an error status view with the glyph being hidden with too small of a height', () => {
+    before(() => browser.url('/#/tests/status-view-tests/height-too-small'));
+
+    shouldTheme({
+      '--terra-status-view-padding-bottom': '10px',
+      '--terra-status-view-padding-top': '10px',
+      '--terra-status-view-title-color': 'purple',
+      '--terra-status-view-title-font-family': 'Impact',
+      '--terra-status-view-title-font-size': '36px',
+      '--terra-status-view-message-color': 'purple',
+      '--terra-status-view-message-font-family': 'Impact',
+      '--terra-status-view-message-font-size': '24px',
+      '--terra-status-view-message-padding-top': '10px',
+      '--terra-status-view-message-text-align': 'left',
+      '--terra-status-view-divider-padding-top': '10px',
+      '--terra-status-view-buttons-flex-direction-small-width': 'column',
+      '--terra-status-view-buttons-padding-top': '10px',
+      '--terra-status-view-button-flex': 'auto',
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Displays a no-data status view with the content centered in its container', () => {
+    before(() => browser.url('/#/tests/status-view-tests/centered'));
+
+    shouldTheme({
+      '--terra-status-view-padding-bottom': '10px',
+      '--terra-status-view-padding-top': '10px',
+      '--terra-status-view-title-color': 'purple',
+      '--terra-status-view-title-font-family': 'Impact',
+      '--terra-status-view-title-font-size': '36px',
+    });
+
+    Terra.should.matchScreenshot({ viewports });
+    Terra.should.beAccessible({ viewports });
+  });
+
+  describe('Displays a no-matching-results status view with the content aligned at the top in its container', () => {
+    before(() => browser.url('/#/tests/status-view-tests/align-top'));
+
+    shouldTheme({
```

Comment:
I don't think we need to test theme properties for every scenario. Just pick a representative scenario (usually a default one) and test the properties on that.

Reply:
Also, I don't think we need to test theming properties for responsive variants unless there is a specific responsive scenario to test.